run_sms_pipeline.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
Can this be 'run_pipeline.sh' now that our pipelines can handle other data sources too?